wear/wear.iml

```diff
@@ -30,6 +31,7 @@
     <output-test url="file://$MODULE_DIR$/build/intermediates/classes/androidTest/xdrip/debug" />
     <exclude-output />
     <content url="file://$MODULE_DIR$">
+<<<<<<< HEAD:wear/wear.iml
```

Comment:
There are merge conflicts here, but I don't think you need to submit this file in any case.